.github/workflows/production.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   deploy-to-staging:
     uses: SumoLogic/sumologic-documentation/.github/workflows/build_and_deploy.yml@main
     with:
-      hostname: https://d3f0cnhsmnxacx.cloudfront.net
+      hostname: https://help.sumologic.com
```

Comment:
@SumoLogic/open-source-collection-team @swiatekm-sumo @mccartney , can you confirm whether this URL should be https://help-opensource.sumologic.com or https://help.sumologic.com?

Reply:
Have some urgent changes to push to prod today -- going to use https://help-opensource.sumologic.com